downstream/assemblies/platform/assembly-aap-migration.adoc

```diff
@@ -0,0 +1,26 @@
+
+ifdef::context[:parent-context: {context}]
+
+[id="aap-migration"]
+= Migrating {PlatformName} to {OperatorPlatform}
+
+:context: aap-migration
+
+[role="_abstract"]
+
+Migrating your {PlatformName} deployment to the {OperatorPlatform} allows you to take advantage of the benefits provided by a Kubernetes native operator, including simplified upgrades and full lifecycle support for your {PlatformName} deployments.
+
```

Comment:
I think that we should point out that this migration guide can be used to migrate the AAP Operator from:
* A VM-based installation of AutomationController or AutomatonHub
* OR an Openshift instance of Ansible Tower 3.8.7 (AAP 1.2.7)

We should also make it clear that if they are also migrating an AutomationHub instance, they will need to repeat the steps of creating a hub secret key and database/postgresql secret, as well as create a separate AutomationHub instance with them specified.  (sections 6.2 and 6.3)

Reply:
Suggested changes complete.